spread/build/fedora/task.yaml

```diff
@@ -41,7 +41,7 @@ execute: |
       --plugin-option=ccache:hashdir=False \
       --plugin-option=ccache:show_stats=True \
       --with debug \
-      --with run_tests \
+      --with check \
```

Comment:
This is not necessary anymore since the default is to run them. You only need to add an option for the non-default case (ie. `--without check`).